File: Programming Basics/For Loop - Lab/09.LeftAndRightSum.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        double leftSum = 0;
        double rightSum = 0;

        for (int i = 0; i < n; i++)
        {
            leftSum += double.Parse(Console.ReadLine());
        }
        for (int j = 0; j < n; j++)
        {
            rightSum += double.Parse(Console.ReadLine());
        }

        if (leftSum == rightSum)
        {
            Console.WriteLine($"Yes, sum = {leftSum}");
        }
        else
        {
            Console.WriteLine($"No, diff = {Math.Abs(leftSum - rightSum)}");
        }
    }
}

File: Programming Basics - Python/For Loop - Lab/09.LeftAndRightSum.py
n = int(input())

left_sum = 0
right_sum = 0

for num in range(0, n):
    left_sum += int(input())

for num in range(0, n):
    right_sum += int(input())

if left_sum == right_sum:
    print(f"Yes, sum = {left_sum}")
else:
    print(f"No, diff = {abs(left_sum - right_sum)}")